crl-release-26.2: go 1.26 build failure

The nightly linux-crossversion test run failed on 00d1f45167c70bfedc2301bc48900557148e26cd (go1.26). Please review the run [here](https://github.com/cockroachdb/pebble/actions/runs/24450428622).

Jira issue: PEBBLE-1406

the json outputs are happening on this branch too
```
--- FAIL: TestLint (76.26s)
    --- FAIL: TestLint/TestRoachVet (33.45s)
        lint_test.go:188: 
            {}
        lint_test.go:188: 
            {}
        lint_test.go:188: 
            {}
        lint_test.go:188: 
            {}
        lint_test.go:188: 
            {}
```

--- a/.github/workflows/nightlies-26.1.yaml
+++ b/.github/workflows/nightlies-26.1.yaml
@@ -33,7 +33,7 @@ jobs:
     with:
       sha: ${{ needs.resolve-sha.outputs.sha }}
       file_issue_branch: ${{ needs.resolve-sha.outputs.branch }}
-      go_version: 1.26
+      go_version: 1.25
 
   s390x:
     needs: resolve-sha
@@ -41,7 +41,7 @@ jobs:
     with:
       sha: ${{ needs.resolve-sha.outputs.sha }}
       file_issue_branch: ${{ needs.resolve-sha.outputs.branch }}
-      go_version: 1.26
+      go_version: 1.25
 
   stress:
     needs: resolve-sha
@@ -49,7 +49,7 @@ jobs:
     with:
       sha: ${{ needs.resolve-sha.outputs.sha }}
       file_issue_branch: ${{ needs.resolve-sha.outputs.branch }}
-      go_version: 1.26
+      go_version: 1.25
 
   instrumented:
     needs: resolve-sha
@@ -57,4 +57,4 @@ jobs:
     with:
       sha: ${{ needs.resolve-sha.outputs.sha }}
       file_issue_branch: ${{ needs.resolve-sha.outputs.branch }}
-      go_version: 1.26
+      go_version: 1.25

--- a/.github/workflows/nightlies-26.2.yaml
+++ b/.github/workflows/nightlies-26.2.yaml
@@ -33,7 +33,7 @@ jobs:
     with:
       sha: ${{ needs.resolve-sha.outputs.sha }}
       file_issue_branch: ${{ needs.resolve-sha.outputs.branch }}
-      go_version: 1.26
+      go_version: 1.25
 
   s390x:
     needs: resolve-sha
@@ -41,7 +41,7 @@ jobs:
     with:
       sha: ${{ needs.resolve-sha.outputs.sha }}
       file_issue_branch: ${{ needs.resolve-sha.outputs.branch }}
-      go_version: 1.26
+      go_version: 1.25
 
   stress:
     needs: resolve-sha
@@ -49,7 +49,7 @@ jobs:
     with:
       sha: ${{ needs.resolve-sha.outputs.sha }}
       file_issue_branch: ${{ needs.resolve-sha.outputs.branch }}
-      go_version: 1.26
+      go_version: 1.25
 
   instrumented:
     needs: resolve-sha
@@ -57,4 +57,4 @@ jobs:
     with:
       sha: ${{ needs.resolve-sha.outputs.sha }}
       file_issue_branch: ${{ needs.resolve-sha.outputs.branch }}
-      go_version: 1.26
+      go_version: 1.25

